apps/src/applab/PhoneFrame.jsx

```diff
@@ -73,48 +70,7 @@ class PhoneFrame extends React.Component {
 }
 
 const styles = {
-  phoneFrame: {
-    display: 'block',
-    height: FRAME_HEIGHT,
-    backgroundColor: color.lighter_gray
-  },
-  phoneFrameDark: {
-    backgroundColor: color.charcoal
-  },
-  phoneFrameTop: {
-    borderTopLeftRadius: RADIUS,
-    borderTopRightRadius: RADIUS
-  },
-  phoneFrameBottom: {
-    borderBottomLeftRadius: RADIUS,
-    borderBottomRightRadius: RADIUS
-  },
-  screenSelector: {
-    marginLeft: 'auto',
-    marginRight: 'auto',
-    paddingTop: (FRAME_HEIGHT - ScreenSelectorStyles.dropdown.height) / 2,
-    width: '80%'
-  },
-  centeredInFrame: {
-    marginLeft: 'auto',
-    marginRight: 'auto',
-    width: '100%',
-    textAlign: 'center',
-    display: 'flex',
-    alignItems: 'center',
-    justifyContent: 'center',
-    height: FRAME_HEIGHT
-  },
-  paused: {
-    color: 'white',
-    fontSize: 20
-  },
-  pauseIcon: {
-    marginRight: 5
-  },
   buttonMinWidth: {
     minWidth: CompletionButtonStyles.phoneFrameButton.minWidth
   }
```

Comment:
these JS styles can't be removed until we refactor the CompletionButton to CSS modules